CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Allow the deletion of specific object versions (gh#aquarist-labs/s3gw#764).
+- Add `S3GW_S3_ADDRESSING_STYLE` environment variable to configure the S3 addressing style (gh#aquarist-labs/s3gw#770). 
```

Comment:
I remember back the days we discussed that we are using this schema because it is specified somewhere in SUSE or openSUSE.

Reply:
I honestly don't remember. I just think that it's not particularly readable as is, and will probably not translate well (or at all) when github parses it. Without the `gh#` I'm fairly certain it will translate that to a link.

Reply:
Done